addons/breathing/functions/fnc_fullHealLocal.sqf

```diff
@@ -28,6 +28,9 @@ _patient setVariable [QGVAR(etco2Monitor), [], true];
 _patient setVariable [QGVAR(breathRate), 15, true];
 _patient setVariable [QGVAR(nasalCannula), false, true];
 
+_patient setVariable [QGVAR(oxygenMaskActive), false, true];
```

Comment:
should heal detach oxymask and make status as 0,0?

Reply:
Nah, because then you'll lose the mask you already have on. Actually, I'll just remove the oxygenMaskActive line so that fullHeals dont fuck with that